studybuddies/src/app/components/construction-page/construction-page.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ConstructionPageComponent } from './construction-page.component';
+
+describe('ConstructionPageComponent', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7167485)
```suggestion
describe("ConstructionPageComponent", () => {
```